docs/source/release/v6.2.0/index.rst

```diff
@@ -16,7 +16,14 @@ Mantid 6.2.0 Release Notes
 
 We are proud to announce version 6.2.0 of Mantid.
 
-**TODO: Add paragraph summarizing big changes**
+This release sees many new algorithms added or improved to extend the functionality of Mantid for users. In addition to this there are some significant new features
+including:-
+
+- Superplot, a new detector widget for the plot window that facilitates over-plotting and the manipulation of overplotted data.
```

Comment:
Can I ask what we mean by detector widget here? The spectra are related to detectors but they don't have to be...

Reply:
I think originally this was decorator, got mistaken during the clean-up. In any case I think it is a too technical term for the end user, so just saying a new widget is fine. I'd suggest:
`Superplot, a new widget that facilitates over-plotting and data comparison with large number of workspaces.`

Reply:
Agreed - I misread the description from the workbench notes but I think @gvardany simplified sentence works much better.